test/framework/openshift.go

```diff
@@ -184,8 +184,8 @@ func WaitAndRetrieveRouteURI(namespace, serviceName string) (string, error) {
 }
 
 // WaitForOnOpenshift is a specific method
-func WaitForOnOpenshift(namespace, display string, timeoutInMin int, condition func() (bool, error)) error {
-	return WaitFor(namespace, display, GetOpenshiftDurationFromTimeInMin(timeoutInMin), condition)
+func WaitForOnOpenshift(namespace, display string, timeoutInMin int, condition func() (bool, error), errorCondition func() (bool, error)) error {
```

Comment:
we could create also a WaitForOnOpenshift(namespace, display, timeoutInMin, condition) method that would directly call this one with NoErrorCondition ?

Reply:
woops, @sutaakar was faster XD

Reply:
We cannot have the same methods with different arguments. I can create another method WaitForOnOpenshiftWithErrorCondition which is a bit verbose bearing in mind that providing an error condition should be a good practice.... but as you prefer ...